web/scss/features/layers.scss

```diff
@@ -386,9 +386,14 @@
   color: #ccc;
 }
 
+.layer_picker_component .modal {
+  max-height: 100vh !important;
```

Comment:
Same thing here about the `!important`.  I would look at how the styles are set up in prod and determine what style changes caused this to break and see if they can be reversed.  That way we can avoid having to add additional styles and forcing them into place with `!important`

Reply:
My bad I totally forgot to fix the initial selection screen on the modal. It should be identical to the way it looks in production now. I went through the layers and facets scss files and tried to remove as many `!important` as I could.